src/settings/Translations.js

```diff
@@ -3,11 +3,45 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Pane } from '@folio/stripes/components';
 
+function renderArray(a) {
+  return a.map(t => {
+    let str;
+    switch (t.type) {
+      case 1: // placeholders for strings, e.g. {foo}
+      case 2: // placeholders with types, e.g. {foo, number}
+        str = `{${t.value}}`;
+        break;
+      case 8: // html elements, e.g. <strong>foo</strong>
+        str = `<${t.value}>${renderArray(t.children)}</${t.value}>`;
+        break;
+      case 0: // simple values without placeholders, e.g. foo
+      default:
+        str = t.value;
+        break;
+    }
+    return str;
+  }).join('');
+}
+
+/**
+ * Raw translation values, i.e. those from .json files, are simple strings.
+ * If the file has been transformed into an AST, the value will be an array;
+ * such values are naively un-AST-ified in a manner that seems to work so far.
+ *
+ * @see https://formatjs.io/docs/guides/advanced-usage#pre-compiling-messages
+ *
+ * @param {*} t
```

Comment:
What is this?

Reply:
It's markup auto-inserted by my editor.
* `@see` is me remembering how javadoc worked 20 years ago in grad school.
* `@param` is VSCode noticing this function takes one param, `t`, of any type, `{*}`. It would more accurately be `{string|array}`.